<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexManager</class>
 <widget class="QDialog" name="IndexManager">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>160</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cache Manager</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="simRadioButton">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>70</y>
     <width>88</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="crawlRadioButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Crawl</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectFile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
